extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -32,10 +34,48 @@ export function getMetadata(request) {
   }
 
   if (matches.length === 0) {
-    return null;
+    exception = getException(request.domain);
+    if (!exception) return null;
+
+    matches = [
+      {
+        pattern: { key: exception.id, name: exception.id },
```

Comment:
would rather use `request.domain` instead of `exception.id`

Reply:
Ok